feat: Add Netlify readme (#1028)
contentful/apps: apps/netlify/README.md

## Local Development

In order to develop Netlify App locally you will need to run locally both the Netlify Contentful 
Application (living in `frontend`), and the Lambda functions (in `lambda`) which handle Netlify build hooks.

Technologies involved are:
* [serverless](https://github.com/serverless/serverless):
to run the Lambda function on your machine
* [ngrok](https://ngrok.com/):
to create a public URL to be used as OAUTH URI

### Launching the frontend
Go to the `frontend` folder and run:
```
npm run start
```

### Launching the lambda
First, start HTTP tunnel with ngrok:
```
ngrok http 8000
```

Enable events and add https ngrok url to the events config on App definition page as follows:
`https://<ngrok url>/app-events`

You would also want to enable Entry and Asset publish/unpublish events:
![Events configuration](images/events-settings.png)

Run lambda with command:
```
npm run serve
```

This script expects the following environment variables to be set:

| Variable                      | Description                                                               |
| ---                           | ---                                                                       |

> | `APP_IDENTITY_PRIVATE_KEY`    | Private key of the key pair generated via App definition page |

Reviewer: Maybe explain how to copy this correctly, i.e: download the private key file from security tab, copy/paste the content of the file to a string as an env variable. I find this to be a tricky part for most people to get

Netlify App
===

## Local Development

> In order to develop Netlify App locally you will need to run locally both the Netlify Contentful 

Reviewer 2: ```suggestion
In order to develop Netlify App locally, you will need to run both the Netlify Contentful 
```

Application (living in `frontend`), and the Lambda functions (in `lambda`) which handle Netlify build hooks.

Technologies involved are:
* [serverless](https://github.com/serverless/serverless):
to run the Lambda function on your machine
* [ngrok](https://ngrok.com/):
to create a public URL to be used as OAUTH URI

### Launching the frontend
Go to the `frontend` folder and run:
```
npm run start
```

### Launching the lambda
First, start HTTP tunnel with ngrok:
```
ngrok http 8000
```

Enable events and add https ngrok url to the events config on App definition page as follows:
`https://<ngrok url>/app-events`

> Go to app's Security tab. Generate key pair and signing secret. These are needed as environment variables (see below). For example you can copy private key contents and paste it as a value to `APP_IDENTITY_PRIVATE_KEY` variable.

Reviewer 2: ```suggestion
Go to the app's Security tab. Generate a key pair and a signing secret. These are needed as environment variables (see below). For example, you can copy the private key and paste it as a value to the `APP_IDENTITY_PRIVATE_KEY` variable.
```

> You would also want to enable Entry and Asset publish/unpublish events:

Reviewer 2: ```suggestion
To receive Entry and Asset publish/unpublish events, configure the Events tab the following:
```
